tests/data/README.md

```diff
@@ -95,3 +95,9 @@ This directory provides test data to demonstrate functionality.
 - Notes
   - `items.geojson` tests pygeoapi's capability to serialize all geometry types for individual collection items in [JSON-LD formats](https://docs.pygeoapi.io/en/latest/configuration.html#linked-data), including GeoSPARQL WKT and schema.org/geo
   - The features represent the range of GeoJSON geometry types, instead of real locations. Additionally, each feature has a uri defined in the properties block
+
+### `USGS Earthquakes`
```

Comment:
I am guessing this is the remote data used in the Socrata tests? tests/data/README.md is for documenting local data housed in the pygeoapi repo proper.

Suggest to include this information in `tests/test_socrata_provider.py`